sdk/tests/conformance2/buffers/uniform-buffers.html

```diff
@@ -82,6 +82,27 @@
     oColor = vec4(UBORed * UBOR, UBOGreen * UBOG, UBOBlue * UBOB, 1.0);
 }
 </script>
+<script id='fshadernamed' type='x-shader/x-fragment'>#version 300 es
+precision mediump float;
+layout(location=0) out vec4 oColor;
+
+uniform UBOData {
+    float Red;
+    float Green;
+    float Blue;
+} UBOA;
+
+uniform UBOData2 {
+    float Red;
+    float Green;
+    float Blue;
+} UBOB;
```

Comment:
Is the second named uniform block here needed in order to provoke the crash? If not, let's consider removing it. If so, let's set it up via the API calls below.